docs/2-search/3-test-search.mdx

```diff
@@ -18,6 +20,11 @@ and look at the results you get.
 
 <Screenshot src="img/screenshots/2-search/3-test-search/2-search-tester.png" alt="The Search Tester" url="https://cloud.mongodb.com" />
 
+:::info
+Some of the fields in the documents (like `title`) may be hidden. You can expand a document to see all of its fields by clicking the arrow button on the right side of the document.
+<img src={useBaseUrl("img/screenshots/2-search/3-test-search/4-expand-document.png")} alt="The expand document button" />
```

Comment:
I tried using the Screenshot component here, but it wasn't displaying correctly inside of the info widget. I went with a regular image instead. If there is a way to get it to display correctly, I'm good with using the Screenshot component.